pangaea/datasets/ai4smallfarms.py

```diff
@@ -9,7 +9,7 @@
 from tifffile import imread
 
 from pangaea.datasets.base import GeoFMDataset
-
+from pangaea.engine.data_preprocessor import BasePreprocessor
```

Comment:
Hi, on this I disagree :)

I see your point, but in deep learning community is very common to have the transformations (i.e. the preprocessing) in the dataloader. 

So I think it will be clear for people who read the code. And in general the code will be cleaner.

For similar reasons, I don't think people who want to add new datasets will forget, especially, if we document it in the readme.

Reply:
In my 10 years of deep learning I've only ever seen it in repositories that were dealing with a single dataset, but it might be that I'm just old-school 😄

Reply:
I rethink about it and we can do a single dataset wrapper (not recursive) to combine a raw dataset and the transforms.